### DISCUSSION TITLE
Podman Network needs to be recreated in version 4.3

### DISCUSSION THREAD

**Discussion by talha110** [containers/podman#17886] [General]
Hi Folks. I was using Podman 4.1.1 a few months back and faced this issue that needs to be recreated - option "parent" has changed. At that time I come to know that this was an issue in Podman and will be fixed in version 4.3. Today I update my Podman on my RHE: offline VM on Version: 4.3.1 and again faced the same issue.  Is there anything else I need to update? 
```
version:
  APIVersion: 4.3.1
  Built: 1675202775
  BuiltTime: Tue Jan 31 17:06:15 2023
  GitCommit: ""
  GoVersion: go1.19.4
  Os: linux
  OsArch: linux/amd64
  Version: 4.3.1
```
Please share the link if this has been already asked or discussed 🙏

**Comment by Luap99** [comment]
I am not quite sure what you are talking about. Can you link the issue?
Also I recommend you test it with v4.4.2 or main to test if the issue is still present in podman.

**Reply by talha110** [reply]
I update the docker-compose version which resolves the above issue. 
As for Podman version 4.4 is not available for CentOS 8 stream, Last release is <html>
<body>


podman-4.3.1-2
--



</body>
</html>
http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/

**Reply by rhatdan** [reply]
@jnovy PTAL

**Reply by jnovy** [reply]
@talha110 We are switching to modularity support for c8s, meaning it will take still a week or more to be able to build modules there reliably. Currently we have problems with building oci-seccomp-pbf-hook [1] due to its dependency on bcc/llvm.

Can you try c9s? There are no modules there and quite fresh podman (4.4.1 at the moment).

[1] https://kojihub.stream.centos.org/kojifiles/work/tasks/8368/2078368/root.log

**Reply by umermala** [reply]
I am currently using podman 4.9.4-rhel version and this issue still persists. Any update on this?
